alchemist/alchemist-cognitive-agents/src/main/kotlin/it/unibo/alchemist/model/cognitiveagents/characteristics/cognitive/BeliefDanger.kt

```diff
@@ -1,11 +1,20 @@
 package it.unibo.alchemist.model.cognitiveagents.characteristics.cognitive
 
-import it.unibo.alchemist.model.interfaces.CognitivePedestrian
-
+/**
+ * The belief of the situation dangerousness.
```

Comment:
Maybe "perception" would be more accurate. Is there a reason why this is called belief? We may want to refactor and call these things "Perceptions". This could become PerceiveDanger. Would it be wrong?

Reply:
The name "belief" comes from Nathalie and Daniel work on the IMPACT model, I've used the same just as a reference but they can be changed easily if you think it's more explicative.

Reply:
Add a note on the documentation of belief explaining the origin of the term. Also, explain it in the wiki documentation.

Reply:
fixed